DI-RAIZEL09/Computer-Store
# feature/Bestseller

Title:

Add Recommendations and ContentForProduct components for homepage

Description:

This PR introduces two new components designed for the homepage:

Recommendations Component:
 ‣Displays recommended products in a grid layout.
 ‣Enhances user engagement by showcasing highlighted items.
 ‣ContentForProduct Component:
 ‣Displays product details and associated spare parts.
 ‣Provides a clear and structured presentation of products.
 
Changes:
 ‣Created Recommendations component.
 ‣Created ContentForProduct component.
 ‣Integrated both components into the homepage layout.

How to Test:
 ‣Run the application and navigate to the homepage.
 ‣Verify that the Recommendations component displays the recommended products correctly.
 ‣Ensure the ContentForProduct component displays the product details and spare parts accurately.

Please review and provide feedback. Thank you!
